src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string;
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    key: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.callbackClickEvent);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.callbackClickEvent);
+  }
+
+  callbackClickEvent = (event: KeyboardEvent) => {
```

Comment:
1 rule in checklist
https://github.com/mate-academy/react_keyboard/blob/master/checklist.md